.github/workflows/image.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     paths-ignore:
       - "**/README.md"
+  workflow_dispatch:
```

Comment:
Don't we need a bit more after `workflow_distpatch` eg https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
Maybe later but I do not see why we need any args for now.

Reply:
ok let's call it a TODO